bot.py

```diff
@@ -5,26 +5,69 @@
 
 client = commands.Bot(command_prefix = "$")
 
-conn = sqlite3.connect('users.db')
+conn = sqlite3.connect(':memory:')
 
 c = conn.cursor()
 
 c.execute("""CREATE TABLE IF NOT EXISTS users (
              irl,

```

Comment:
Can you put data types on these? eg `VARCHAR(64)`

Reply:
In this case, I'd suggest `TEXT`.